extra/liquidexport

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python3
+"""liquidexport – host a metrics HTTP endpoint with Prometheus formatted data from liquidctl
+
+This is an experimental script that collects stats from liquidctl and exposes them as a http://localhost:8098/metrics
+endpoint in the Prometheus text format.
+See: https://prometheus.io/docs/instrumenting/exposition_formats/#text-format-example
+
+Example metric with labels:
+# HELP liquidctl liquidctl exported metrics
+# TYPE liquidctl gauge
+liquidctl{device="NZXT Kraken X (X42, X52, X62 or X72)",sensor="liquid_temperature",unit="°C"} 33.6
+
+Usage:
+  liquidexport [options]
+  liquidexport --help
+  liquidexport --version
+
+Options:
+  --interval <seconds>    Update interval in seconds [default: 2]
+  --hid <module>          Override API for USB HIDs: usb, hid or hidraw
```

Comment:
It is no longer necessary to forward `--hid` since it is ignored since 1.4.0.

Reply:
Removing the `--hid` option lead to an error since `liquidctl/cli.py` still cares about `--hid`. 
```python
Traceback (most recent call last):
  File "liquidexport", line 97, in <module>
    frwd = _borrow._make_opts(args)
  File "/mnt/c/Users/alexb/code/play/liquidctl/src/liquidwslenv/lib/python3.8/site-packages/liquidctl/cli.py", line 220, in _make_opts
    if args['--hid']:
KeyError: '--hid'
```

I piggybacked on the same `args` forwarding boilerplate from `liquiddump` which passes them onto `liquidctl.cli`:
```python
import liquidctl.cli as _borrow
if __name__ == '__main__':
    args = docopt(__doc__, version='0.1.0')
    frwd = _borrow._make_opts(args)
    devs = list(find_liquidctl_devices(**frwd))
```

If I remove all device filtering then I can avoid this issue which seems fine for this tool.